psl-core/src/test/java/org/linqs/psl/evaluation/statistics/RankingEvaluatorTest.java

```diff
@@ -29,6 +43,77 @@ protected RankingEvaluator getComputer() {
         return new RankingEvaluator();
     }
 
+    @Before
+    public void setUp() {
```

Comment:
Do all the tests in this file need this setup?
Are we just complicating the existing tests?

If we are going to move the new functionality out to another `Evaluator`, then we can just make a new test and this will be a non-issue.